.travis.yml

```diff
@@ -11,18 +11,26 @@ env:
     - TVOS_FRAMEWORK_SCHEME=Hyperspace_tvOS
     - WATCHOS_FRAMEWORK_SCHEME=Hyperspace_watchOS
   matrix:
-    - DESTINATION="platform=iOS Simulator,OS=11.2,name=iPhone X"                        SCHEME="$IOS_FRAMEWORK_SCHEME"      RUN_TESTS="YES" POD_LINT="YES"
+    - DESTINATION="platform=iOS Simulator,OS=11.2,name=iPhone X"                        SCHEME="$IOS_FRAMEWORK_SCHEME"      RUN_TESTS="YES" POD_LINT="YES"  RUN_DANGER="YES"
```

Comment:
I don't know if this is just GitHub, but the spacing looks off here.

Reply:
It's because of the long name for the Watch simulator a few lines below. It looks better when you're seeing all 3 platforms together:
![image](https://user-images.githubusercontent.com/1720304/50717243-84e98980-104b-11e9-97a5-9215da8b27c1.png)

Reply:
Makes sense. Will merge as soon as CI passes.